src/components/tree-views-container.jsx

```diff
@@ -474,18 +482,6 @@ const TreeViewsContainer = () => {
                 ) {
                     dispatch(setSelectedDirectory(null));
                 }
-                if (
```

Comment:
we still have the bug when renaming root directories, I think that we need to keep this code